config/fxdata/terrain.cfg

```diff
@@ -2,195 +2,861 @@
 ; file version 0.32, by Mefistotelis
 
 [common]
-SlabsCount = 54
+SlabsCount = 55
 RoomsCount = 17
 
 ; Slabs configuration
 
 [slab0]
 Name = HARD
 TooltipTextID = 324
+BlockFlagsHeight = 4
+BlockHealthIndex = 0
+BlockFlags = BLOCKING FILLED
```

Comment:
What are all the possible BlockFlags?

Reply:
By this question I mean: Could you comment the cfg to list all the possible flag. Same as with the NoBlockFlags.